extensions/amp-timeago/1.0/timeago.js

```diff
@@ -49,7 +53,20 @@ export function Timeago(props) {
   }, [datetime, locale, cutoff, cutoffText]);
 
   useResourcesNotify();
-  return createElement('time', {datetime, ref}, timestamp);
+
+  return (
+    <ContainWrapper
+      as="time"
+      size={containSize}
+      layout={true}
+      paint={true}
+      contentRef={ref}
+      datetime={datetime}
+      {...rest}
```

Comment:
I'm not sure about this. Does this mean the publisher can override i.e. `contentRef` here before `ContainWrapper` is instantiated?

Reply:
The `contentRef` is not in the allowed props of the `TimeAgo`. But do you think that concern would be addressed if I were to put `{...rest}` before all other props?

Reply:
I moved it up top. Actually I always thought that prop spread was automatically always first. But it looks like that's not the case. So this way is safer.